cpu/stm32/periph/Kconfig

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_PERIPH
+    bool
+    default y
+    select MODULE_XTIMER if MODULE_PERIPH_USBDEV
+    help
+        stm32 common peripheral code.
+
+config MODULE_PERIPH_UART_NONBLOCKING
+    depends on HAS_PERIPH_UART_NONBLOCKING
+    depends on MODULE_PERIPH_UART
+    select MODULE_TSRB
+
+config MODULE_PERIPH_CAN
+    bool
+    depends on HAS_PERIPH_CAN
+    depends on HAS_PERIPH_GPIO
+    depends on HAS_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_I2C
+    help
+        stm32 can bus peripheral.
```

Comment:
No sure about the help string, how is this resolved against the other definition?